Assignment2/Sixtrax-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,53 @@
+{
+	"@context":{
+			"class":"http://www.example.org/Class#",
+      		"element":"http://www.example.org/Elements#",
+			"pr":"http://www.example.org/Properties#",
+      		"msr":"http://www.example.org/Measurements#",
+      		"user":"http://www.example.org/Users#",
+			"xsd":"http://www.w3.org/2001/XMLSchema#",
+			"Class01":"class:Class01",
+      		"includes":"pr:includes",
+			"hasMeasurement":"pr:hasMeasurement",
+			"hasOwner":"pr:hasOwner",
+			"hasTemperature":"pr:hasTemperature",
+			"atTime":"pr:atTime",
+			"hasName":"pr:hasName",
+			"Sensor029":"element:Sensor029",
+			"Computer101":"element:Computer101",
+			"mMeasurement8401":"msr:Measurement8401",
+			"User10A":"user:User10A"
+	},
+	"Class01":{
+		"@id":"http://www.example.org/Class01",
+		"@type":"class:Class01",
+		"Sensor029":{
+			"@id":"http://www.example.org/Class01/Sensor029",
+			"@type":"element:Sensor029",
+			"Measurement8401":{
+				"@id":"http://www.example.org/Class01/Sensor029/Measurement8401",
+				"@type":"msr:Measurement8401",
+				"hasTemperature":{
+					"@value":"29",
+					"@type":"xsd:integer"
+				},
+				"atTime":{
+					"@value":"2010-06-12T12:00:12",
+					"@type":"xsd:dateTime"
+				}
+			}
+		},
+		"Computer101":{
+			"@id":"http://www.example.org/Class01/Computer101",
+          	"@type":"element:Computer101",
+			"User10A":{
+				"@id":"http://www.example.org/Class01/Computer101/User10A",
+				"@type":"user:User10A",
+				"hasName":{
+					"@value":"Pedro",
+					"@type":"xsd:string"
+				}
+			}
+		}
+	}		
+} 
```

Comment:
You define  "includes", "hasOwner", "hasMeasurement" attributes in @context, but you don't define them in your data.